README.md

```diff
@@ -1,15 +1,96 @@
+  - [macOS](#macos)
+  - [Windows](#windows)
+  - [Linux](#linux)
+- [🛠️ Build from Source](#-build-from-source)
+- [🤝 Contributing](#-contributing)
+- [📚 Documentation](#documentation)
+
+
+## 🚀 Download & Install
+
+Get the latest release for your platform:
+
+- **Windows**: [Download EXE](https://github.com/status-im/status-desktop/releases/latest)
+- **Linux**: [Download Tarball](https://github.com/status-im/status-desktop/releases/latest)
+- **macOS**: [Download DMG](https://github.com/status-im/status-desktop/releases/latest)
+
+### Windows
+
+1. [Download](https://github.com/status-im/status-desktop/releases/latest) the `.exe` file.
+2. Run the `.exe` installer.
+3. Launch the installed app 🎉
+
+### Linux
+
+1. [Download](https://github.com/status-im/status-desktop/releases/latest) the `.tar.gz` file.
+2. Extract the tarball file (replace `*` with the version)
+    ```shell
+    cd ~/Downloads
+    tar xvf StatusIm-Desktop-*.tar.gz
+    ```
+3. Optionally, [verify the Status app GPG signature](https://status.app/help/getting-started/download-status-for-linux#verify-the-status-app-gpg-signature).
+4. Go to the directory where it extracted and double-click the AppImage file 🎉
+
+### macOS
+
+1. [Download](https://github.com/status-im/status-desktop/releases/latest) the `.dmg` file.
+2. Open it and drag `Status.app` to the Applications folder.
+3. Open the Applications folder and double-click the Status icon 🎉
+
+
+## 🛠️ Build from Source
+
+<!-- TODO Move building docs to Github -->
+
+To build Status Desktop from source, follow the instructions specific to your operating system. Detailed build instructions are available in the [official documentation](https://zealous-polka-dc7.notion.site/Building-ca1db4fb3baf4f15bab8da717832b743?pvs=74).
+
+
+## 🤝 Contributing
+
+We welcome contributions from the community! To get started:
+
+<!-- TODO Improve the conributing guide to be more about how to contribute -->
+<!-- rename the current one to a style guide -->
+<!-- TODO add guide on how to translate -->
+<!-- TODO Create a guide per persona in the contributing guide -->
+
+- Read our [Contributing Guidelines](CONTRIBUTING.md).
+- Check out [good first issues](https://github.com/status-im/status-desktop/contribute) to get involved.
+- Join the `#feedback-desktop` channel on [Status](hhttps://status.app/cc/G-EAAORobqgnsUPSVCLaSJr855iXTIdQiY1Q0ckBe8dWWEBpUAs9s8DTjWEpvsmpE83Izx1JWQuZrWWKUoxiXCwdtB-wPBzyvv_n9a0F61xTaPZE7BEJDC7Ly_WcmQ4tHRAKnPfXE_JUtEX_3NhnXQN0eh4ue0D77dWvaDpDrSi0U0CaGLZ-pqD_iV0z9RMFE2LKulDZdwL40etJ8lxjyTFoxS0lUhdWKinIOk8qBmJJpCmsqMrSklEU#zQ3shZeEJqTC1xhGUjxuS4rtHSrhJ8vUYp64v6qWkLpvdy9L9).
+
+
+## 📚 Documentation
+
+<!-- TODO Add architecture documentation -->
+
+- [User Help documentation](https://status.app/help)
+- [Installation Guide](https://status.app/help/getting-started#for-new-users)
+- [Building from Source](https://zealous-polka-dc7.notion.site/Building-ca1db4fb3baf4f15bab8da717832b743?pvs=74)
```

Comment:
Is that the correct link now?

Reply:
It is for now. once I merge the BUILDING.md doc, I'll change the link :smile: